engine/src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala

```diff
@@ -156,7 +157,10 @@ trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
     // This may be called multiple times with the same inputs.  Calling this twice with the same
     // parameters should yield the same return value.
     val rootDir = path.toAbsolutePath(fileSystems)
-    if (!rootDir.exists) rootDir.toFile.mkdirs()
+    Try(rootDir.toFile) match {
+      case Success(javaFile) if !javaFile.exists => javaFile.mkdirs
+      case _ =>
+    }
```

Comment:
Use nio's [`Files.createDirectories(rootDir)`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createDirectories-java.nio.file.Path-java.nio.file.attribute.FileAttribute...-), or if a class is already using [`better.files`](https://github.com/pathikrit/better-files#better-files----), [`rootDir.createDirectories()`](https://github.com/pathikrit/better-files/blob/v2.13.0/core/src/main/scala/better/files/File.scala#L85).

In general, all usage of `java.io.File` should (probably) be removed from the cromwell ecosystem to ensure that our custom `java.nio.file.spi.FileSystemProvider` works.